libmscore/edit.cpp

```diff
@@ -4829,13 +4829,13 @@ void Score::undoAddElement(Element* element)
                               sm = cr2->staffIdx() - cr1->staffIdx();
                         Chord* c1 = findLinkedChord(cr1, score->staff(staffIdx));
                         Chord* c2 = findLinkedChord(cr2, score->staff(staffIdx + sm));
-                        Note* nn1 = c1->findNote(n1->pitch(), n1->unisonIndex());
+                        Note* nn1 = c1 ? c1->findNote(n1->pitch(), n1->unisonIndex()) : 0;
```

Comment:
Please use `nullptr`, same for the rest. See the [code style documentation](https://github.com/musescore/Documentation/blob/master/WorkflowAndGuidelines/CodeGuidelines.md).

Reply:
that's the new master code style though...

Reply:
0 is a perfectly acceptable value for a pointer

Reply:
I didn't want to make unnecessary changes (the line below) but still want to fit the new code to the existing one.